service-front/module/Application/view/application/authenticated/lpa/status/index.twig

```diff
@@ -1,4 +1,4 @@
-{% extends 'layout/twig/layout.twig' %}
+{% extends 'layout/layout.twig' %}
```

Comment:
see comparison to other comment

Reply:
I shifted all of the layout twig templates out of the folder called "twig", which was just pointless and another thing to click through when looking for files. I moved those templates up a level to the parent of "twig" and got rid of "twig". Not strictly necessary but annoys the hell out of me when working on these templates :)

Reply:
indeed - thats all good, just that the `twig` folder is still referenced in a number of places above.

Reply:
I think I caught all the references to `layout/twig` but please shout if you can see one I've missed.